Role of scope parameter in the Issuance flow (#631)
italia/eid-wallet-it-docs: docs/en/pid-eaa-issuance.rst

      - A method that was used to derive **code challenge**. It MUST be set to ``S256``.
      - :rfc:`7636#section-4.3`.
    * - **scope**

>       - JSON String. String specifying a unique identifier of the Credential regardless of its format. It MUST be mapped in the `credential_configurations_supported` metadata claim of the Credential Issuer. Unique identifier value MUST match the `credential_unique_id` parameter of the Digital Credentials Catalogue. For example, in the case of the PID, it may be set to ``PersonIdentificationData`` while in case of mobile driving licence ``mDL``. Since it MAY be multivalued, when this occurs each value MUST be separated by a space.

Reviewer: please at least define Digital Credentials Catalogue within the defined terms, for now

Author: This definition will be provided in the PR relating to the Digital Credentials Catalogue.

      - :rfc:`6749`
    * - **authorization_details**
      - Array of JSON Objects. Each JSON Object MUST include the following claims:

            - **type**: it MUST be set to ``openid_credential``,

>             - **credential_configuration_id**: JSON String. String specifying a unique identifier of the Credential in a specific format that MUST be mapped in the `credential_configurations_supported` metadata claim of the Credential Issuer. For example, in the case of the PID, it may be set to ``dc_sd_jwt_PersonIdentificationData`` for PID in SD-JWT VC format, in the case of mobile driving licence in SD-JWT VC format ``dc_sd_jwt_mDL`` and ``mso_mdoc_mDL`` in case of mobile driving license in mdoc format.

Reviewer: ```suggestion
            - **credential_configuration_id**: JSON String. String specifying a unique identifier of the Credential in a specific format that MUST be mapped in the `credential_configurations_supported` metadata claim of the Credential Issuer. For instance,``dc_sd_jwt_PersonIdentificationData`` can be used for PID in SD-JWT VC format, ``dc_sd_jwt_mDL`` for mobile driving licence in SD-JWT VC format and ``mso_mdoc_mDL`` for mobile driving license in mdoc format.
```
